test/run_tests.sh

```diff
@@ -30,39 +30,44 @@ export TRIM_GRAPH_CHECK_FREQUENCY=$GRAPH_CHECK_FREQUENCY
 export TORCH_TEST_DEVICES="$CDIR/pytorch_test_base.py"
 export PYTORCH_TEST_WITH_SLOW=1
 
+function run_test {
+  "$@"
+  # For reasons at this time unknow, only on FB CircleCI, exactly when switching
+  # between these two tests, it seems like a rapid cleanup+init sequence of
```

Comment:
Maybe changed `these two tests` to `tests` since now we do sleep for all tests.

Reply:
Good catch!